static/main.js

```diff
@@ -0,0 +1,59 @@
+$(document).ready(function(){
+    $(".hide-on-start").hide();
+
+    $('#edit-link').on('click', function(event){
+        event.preventDefault();
+
+        var id = $('#form-id').val();
+        var title = $('#form-title').val();
+
+        $.ajax({
+            method: "GET",
+            url: "/edit/" + id + "/" + title
+        }).done(function(response){
+            $("#form-title").val();
+            $('#form-text').val();
+            $('#twitter').hide();
+            $('.entry').hide();
+            $('#edit-form-container').show();
+        }).fail(function(){
+            alert("error");
+        });
+    });
+
+    $('#update-link').on('click', function(event){
+        event.preventDefault();
+
+        var id = $('#form-id').val();
+        var title = $('#form-title').val();
+        var text = $('#form-text').val();
+
+        $.ajax({
+            method: 'POST',
+            url: '/edit/' + id + '/' + title,
+            data: {id: id, text: text, title: title},
```

Comment:
Since you are already passing the `id` in the URL, having it here in form data is a bit redundant.  It does no harm, but also gives no advantage.